yapapi/golem.py

```diff
@@ -63,23 +64,44 @@ class Golem:
     certain, discrete phases of a lifetime of each service instance - startup, running and
     shutdown.
 
-    As `Golem`'s job includes tracking and executing payments for activities spawned by
-    either mode of operation, it's usually good to have just one instance of `Golem` active
-    at any given time.
+    Internally, `Golem`'s job includes running the engine which takes care of first finding the
+    providers interested in the jobs the requestors want to execute, then negotiating agreements
+    with them and facilitating the execution of those jobs and lastly, processing payments. For this
+    reason, it's usually good to have just one instance of `Golem` active at any given time.
```

Comment:
```suggestion
    reason, it's usually good to have just one instance of `Golem` operative at any given time.
```

There will be a property `Golem.operative` in https://github.com/golemfactory/yapapi/pull/644 so I think it would match? Or maybe we should change there, I've no opinion on this.